### DISCUSSION TITLE
Icon request: artificial-intelligence

### DISCUSSION THREAD

**Discussion by pietro-saccani** [FortAwesome/Font-Awesome#21193] [Icon Requests]
I think it would be useful to have an icon of, like, a brain made out of a distributed network? Or something else that immediately recalls the concept of AI.
The name doesn't have to be "artificial-intelligence", you can change it if you find a better one. For example, would just "ai" be ok or would it be too short?
This isn't a request for a concrete object but I don't think it's too abstract to be represented, as an "artificial" brain recalls it pretty easily.

**What can it be used for?**
It could be used to point out that a certain feature uses AI technology. 

**Example image (optional)**
Example 1: https://i.imgur.com/MGm3sjl.png
Example 2: https://i.imgur.com/Ximstaz.jpg

Thank you.

**Comment by ljm22** [comment]
I'd like to see  variants such as

Artificial Intelligence (brain made of circuits)
Automated Reasoning/Machine Learning (brain made of cogs)
Human (brain made of... well... brain)
Human+AI {half and half)}

But needs

There is already robot user and human user of course. There is already brain and cogs. But would be nice for them to have a strong design relationship.
